users/templates/partials/display_messages.html

```diff
@@ -0,0 +1,7 @@
+{% if messages %}
+    <ul class="messages" style="list-style-type: none;">
+        {% for message in messages %}
+            <li><font color="green"><b>{{ message }}</b></font></li>
```

Comment:
Wouldn't it be better to move this formatting to CSS?

Reply:
Definitely! :) Thanks I didn't see that